app/src/main/res/values/strings.xml

```diff
@@ -39,6 +39,10 @@
     <string name="pref_header_cloud_storage">Cloud Storage</string>
     <string name="auto_configure_title">Auto configure</string>
     <string name="prefs_auto_config_summary">Auto configure using a barcode.</string>
+    <string name="auto_config_cam_title">Scanner</string>
+    <string name="auto_config_cam_summary">Scan a QR code with your mobile (recommended)</string>
```

Comment:
shouldn't you mention "camera" here?

Reply:
I spent some time with ChatGPT for these.  But, I see your point.
The problem is if I focus on the meaning of the sentence, saying scan with your camera, it may be taken as take a camera, not your phone, out of your pocket and use it to scan.

What if I change that to the following?

Scan a QR code with your mobile camera

Reply:
Sorry, I even confused myself.  I will think about this and improve the strings.

Reply:
How about this?

Title: Scanner (camera)
Summary: Scan a QR code with your mobile camera